exercises/guillaume_communie/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * Skeleton main routine
+ */
+
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <string>
+#include <cctype>
+#include <iomanip>
+#include <set>
+
+const std::string RESULTS_FILENAME("word_counts.data");
+
+/**
+ * Print the program usage.
+ */
+void usage(void) {
```

Comment:
I don't think you have to specify void for the argument list, you can as well leave it blank.